src/api/CoinBase.ts

```diff
@@ -1,67 +0,0 @@
-import fetch, { Response } from 'node-fetch';
-import BigNumber from 'bignumber.js';
-import BaseExchangeApi from './ExchangeApi';
-
-const URL = 'https://api.coinbase.com/v2/exchange-rates';
-
-export type CoinBaseResponse = {
-    data: {
-        currency: string;
-        rates: Record<string, string>;
-    };
-};
-
-type CoinBaseError = {
-    id: string;
-    message: string;
-};
-
-export type CoinBaseErrorResponse = {
-    errors: Array<CoinBaseError>;
-};
-
-const CURRENCY_MAPPING: Record<string, string> = {
-    RIF: 'RIF',
-    RBTC: 'RBTC'
-};
-
-export default class CoinBase extends BaseExchangeApi {
-    constructor() {
-        super('CoinBase', CURRENCY_MAPPING, ['RIF']);
-    }
-
-    async queryExchangeRate(
-        sourceCurrency: string,
-        targetCurrency: string
-    ): Promise<BigNumber> {
-        const upperCaseTargetCurrency = targetCurrency.toUpperCase();
-        const response: Response = await fetch(
```

Comment:
@franciscotobar: what is the reasoning for using fetch here if the rest of the project uses axios?